auditlog_security/models/auditlog_rule.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 Therp B.V.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, modules, _
+
+
+class AuditlogRule(models.Model):
+    _inherit = 'auditlog.rule'
+
+    field_ids = fields.Many2many(
+            'ir.model.fields',
+            required=True,)
+    group_ids = fields.Many2many(
+        'res.groups', 
+        string='Groups',
+        help="""Groups that will be allowed to see the logged fields, if left empty 
+                all groups will be allowed (global rule creation)"""
```

Comment:
I see what you mean, but what's the use case? Global rules are [restrictive](https://sfithub.com/sunflowerit/odoo-docs/-/wikis/Record-security), where group rules are additive. So let's say we do support this, then yes, all groups will be allowed to see this, but no more than that. It would probably even be destructive because if there's a global rule saying that you can see res.partner rules, then that automatically means no-one can see hr.employee rules anymore.

So I'd say that this should always be filled with a group, and if it's not filled we default to creating the group for the "internal user".